QA_Requests/MultiSelectPerf/hq_workflows.py

```diff
@@ -0,0 +1,135 @@
+import time
+
+from selenium.common import NoSuchElementException, TimeoutException
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support.wait import WebDriverWait
+
+from HQSmokeTests.testPages.base.base_page import BasePage
+from utilities.decorators import timer
+
+
+class AppCreationPage(BasePage):
```

Comment:
I think this class should be in the testPage directory as it doesnot have any test method

Reply:
I don't think this would require a directory structure -- the reason being we do not have multiple files, and I plan to move all except `hq_workflows.py` and `generate_data_capture_readings.py` to common utilities as a part of https://dimagi-dev.atlassian.net/browse/QA-4401